app/policies/founder_policy.rb

```diff
@@ -3,7 +3,7 @@ def report?
     return false if user.blank?
 
     # School admins can view student profiles.
```

Comment:
@a-c-sreedhar-reddy Update the comment as well.

Reply:
@mahesh-krishnakumar I am new to rails. user would be blank when there is an unauthenticated request right?

Reply:
The feedback here was to just remove the comment. But coming to your question, yes `user` would be blank, but we have a guard-clause already to handle the same.